### DISCUSSION TITLE
Using and loving

### DISCUSSION THREAD

**Discussion by rohit-sagar256** [themeselection/sneat-bootstrap-html-admin-template-free#8] [Show and tell]
I am not a frontend guy and  have been working on a project for four months using **[sneat dashboard theme](https://github.com/themeselection/sneat-html-admin-template-free)**  and so far I m very happy that I have chosen **[sneat dashboard theme](https://github.com/themeselection/sneat-html-admin-template-free)** , I m very thankful to themeselection devs as well for making this good looking and awesome theme. 
This theme helped me a lot making the dashboard for my project. Thank you very much ThemeSelection Devs 👏.
<img width="946" alt="Screenshot 2022-08-19 184752" src="https://user-images.githubusercontent.com/70800042/185628421-c5a04322-5126-40c8-9d54-22de69e0a22f.png">

**Comment by rbhargava25** [comment]
Can you please assist me how can I start using this theme for my angular project ? Thanks in advance.
